src/libclipper/src/rpc_service.cpp

```diff
@@ -43,21 +44,28 @@ RPCService::RPCService()
 RPCService::~RPCService() { stop(); }
 
 void RPCService::start(const string ip, const int port) {
+  if(active_.load(std::memory_order_seq_cst)) {
+    throw std::runtime_error("Attempted to start RPC Service when it is already running!");
+  }
   const string address = "tcp://" + ip + ":" + std::to_string(port);
-  active_ = true;
+  active_.store(true, std::memory_order_seq_cst);
```

Comment:
Actually on second thought, let's use the assignment operator.

s/active.store(true,...);/active_ = true;